src/hyp3_isce2/logging.py

```diff
@@ -0,0 +1,13 @@
+import logging
+import sys
+
+
+def configure_root_logger() -> None:
+    # isce2 sets the root logger level to DEBUG, so we re-configure it here.
+    # See: https://github.com/ASFHyP3/hyp3-isce2/issues/53
```

Comment:
Maing this comment as a docstring makes it easier for people to access it.
```suggestion
def configure_root_logger() -> None:
    """Re-configures the root level logger since ISCE2 sets it to DEBUG.
    
    See: https://github.com/ASFHyP3/hyp3-isce2/issues/53 for context.
    """
```